index.js

```diff
@@ -0,0 +1,115 @@
+//! Project Slider 1 
```

Comment:
Great use of vanilla JS for the slider effect - simple, but really adds to the UX to have some interaction.